packages/dvilela/skills/memeooorr_abci/models.py

```diff
@@ -112,3 +114,40 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
         )
 
         super().__init__(*args, **kwargs)
+
+
+def get_store_path(kwargs: dict) -> Path:
+    """Get the path of the store."""
+    path = kwargs.get("store_path", "")
+    if not path:
+        msg = "The path to the store must be provided as a keyword argument."
+        raise ValueError(msg)
+
+    # check if the path exists, and we can write to it
+    if (
+        not os.path.isdir(path)
+        or not os.access(path, os.W_OK)
+        or not os.access(path, os.R_OK)
+    ):
+        msg = f"The store path {path!r} is not a directory or is not writable."
+        raise ValueError(msg)
+
+    return Path(path)
+class StakingParams(BaseParams):
```

Comment:
Akshat you need to merge these params into our alreeady existing params class. So move these params into our class and delete  StakkingParams